docker-compose.yml

```diff
@@ -19,6 +19,9 @@ services:
   postgres:
     image: chrisnharvey/magiclamp-postgres:${MAGICLAMP_VERSION:-1.5}
 
+    volumes:
+      - pg_data:/var/lib/postgresql/data/pgdata:rw${VOLUME_FLAGS}
+
```

Comment:
I think adding this line actually solves a bug, nice catch! ;)

Would be cool if you could cherry-pick this into a new PR that targets `master`.